### PR TITLE
[stable/aerospike] Add load balancer source ranges

### PR DESCRIPTION
**Pull request by FlavioF**
#### What this PR does / why we need it:
It add support to setup k8s service [loadBalancerSourceRanges](https://kubernetes.io/docs/tasks/access-application-cluster/configure-cloud-provider-firewall/) in order to restrict access for LoadBalancer Service.

#### Checklist
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: stable/aerospike/Chart.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ keywords:
   - aerospike
   - big-data
 home: http://aerospike.com
-version: 0.1.7
+version: 0.1.8
```
Comment (paulczar) [inline stable/aerospike/Chart.yaml]: should be a minor release change `0.2.0` rather than a patch.

**Review by paulczar — CHANGES_REQUESTED** [review]

**Comment by FlavioF** [comment]
Done @paulczar

**Pull request merged**